src/Storage/Storage.Management/StorageAccount/SetAzureStorageAccount.cs

```diff
@@ -239,8 +241,8 @@ public bool EnableAzureActiveDirectoryDomainServicesForFile
 
         [Parameter(Mandatory = false, HelpMessage = "Routing Choice defines the kind of network routing opted by the user. Possible values include: 'MicrosoftRouting', 'InternetRouting'")]
         [ValidateSet(
-            Microsoft.Azure.Management.Storage.Models.RoutingChoice.MicrosoftRouting,
-            Microsoft.Azure.Management.Storage.Models.RoutingChoice.InternetRouting,
+            RoutingChoiceType.MicrosoftRouting,
+            RoutingChoiceType.InternalRouting,
```

Comment:
Should be "InternetRouting" instead of "InternalRouting"

Reply:
Done